app/packs/review/_components/PageContent.tsx

```diff
@@ -1,33 +1,31 @@
 'use client'
 
 import { JsonValue } from '@prisma/client/runtime/library'
-import GlowingHeading from '@/components/GlowingHeading'
 import UnreviewedPack from './UnreviewedPack'
+import { useRouter } from 'next/navigation'
 import { PackType } from '@prisma/client'
 import { useState } from 'react'
 
 type UnreviedPackData = {
 	type: PackType
 	id: string
 	name: string
-	pending: boolean
 	description: string
-	denied: boolean
+	status: 'pending' | 'approved' | 'resubmit_pending' | 'denied'
```

Comment:
This should use the Status enum instead of typing each. (don't forget to import it)
```suggestion
	status: Status
```